packages/contracts/test/20_integration-testing/21_deployment.ts

```diff
@@ -97,17 +98,26 @@ async function fixture(): Promise<FixtureResult> {
 
   const [deployer] = await ethers.getSigners();
 
-  // Plugin repo registry
-  const pluginRepoRegistry = PluginRepoRegistry__factory.connect(
-    getLatestNetworkDeployment(getNetworkNameByAlias(productionNetworkName)!)!
-      .PluginRepoRegistryProxy.address,
-    deployer
-  );
-
+  // Plugin Repo
   const {pluginRepo, ensDomain} = await findPluginRepo(env);
   if (pluginRepo === null) {
     throw `PluginRepo '${ensDomain}' does not exist yet.`;
   }
 
+  const network = getNetworkNameByAlias(productionNetworkName);
```

Comment:
If you really wanted to keep the verbosity down just extract into a helper

Reply:
My thinking was that such a helper should sit on the osx-commons-sdk.